init/prepare.sh

```diff
@@ -44,6 +44,12 @@ pushd $SOURCE_DIR/ndt
     export LDFLAGS="-L$BUILD_DIR/build/lib"
     ./bootstrap
     ./configure --enable-fakewww --prefix=$BUILD_DIR/build
+    # Run unit tests on source before making and installing
+    make check
+    if [[ $? -ne 0 ]]; then
```

Comment:
Thanks for pointing that out, I hadn't inspected the top of the file to see what mode bash was operating in! I've implemented your suggestion in the latest commit.